docs/style/disambiguation.md

```diff
@@ -7,17 +7,17 @@ should be written in English and kept fairly short.
 
 ## When to use disambiguation
 
-1. If two or more entities of the same type contain an identical name.
-<br/><br/>
+1.  If two or more entities of the same type contain an identical name.
+    <br/><br/>
 
-2. If two or more entities of the same type contain names where homonyms,
+2.  If two or more entities of the same type contain names where homonyms,
     grammatical articles, punctuation or plurality differ.
 
 ## When *not* to use disambiguation
 
-1. To provide a description of an entity that is not ambiguous. Use
+1.  To provide a description of an entity that is not ambiguous. Use
     annotations to provide such descriptions.
-<br/><br/>
+    <br/><br/>
 
 2. To distinguish entities of different types. Existing cues in the user
```

Comment:
Are these double linebreaks really required here still?
Layout issues with the spacing between list items should not have to be fixed that way, ideally CSS should be used.

Also, in case you have added the second space after 1. to have a consistent indentation of four characters for the whole block of text, you forgot to this for 2. in this section 😏

Reply:
The intent is to provide line breaks between numbered list items, to basically be consistent with those later in the doc. It's a minor nit. I did add the spacing intentionally to just align text. I agree, would be best to solve this in CSS, but that looks like a non-trivial undertaking.